cdap-app-fabric/src/main/java/co/cask/cdap/gateway/handlers/ProgramLifecycleHttpHandler.java

```diff
@@ -491,8 +491,7 @@ public void getProgramRuntimeArgs(HttpRequest request, HttpResponder responder,
                                     @PathParam("namespace-id") String namespaceId,
                                     @PathParam("app-name") String appName,
                                     @PathParam("program-type") String type,
-                                    @PathParam("program-name") String programName) throws BadRequestException,
-    NotImplementedException, NotFoundException, UnauthorizedException {
+                                    @PathParam("program-name") String programName) throws Exception {
```

Comment:
Since `isVisible` method will throw Exception so I changed the individual exceptions